src/odemis/odemisd/test/yaml-merger-combination-realistic-startup-file.yaml

```diff
@@ -0,0 +1,67 @@
+# Microscope file which uses multiple parents for a component:
+# * one parent (SEM Scan Interface) creates the component (CL PMT)
+# * one parent (CL Detector) uses the component
+
+SPARC2: {
+    class: Microscope,
+    role: sparc2,
+    children: ["SEM E-beam", "CL Detector"],
+}
+
+"SEM Scan Interface": !include yaml-merger-combination-SEM-Scan-Interface.yaml
+
+"CL Detector": {
+    class: pmtctrl.PMT,
+    role: cl-detector,
+    init: {
+        # Add via !inlcude
+        settle_time: [0.1, 1, 3],
+        test_input: "a"
```

Comment:
Shouldn't this file be filled by !include and !extend? It's identical to the -expected version, so it's obviously going to pass...

Reply:
That is what is meant with [this comment](https://github.com/delmic/odemis/pull/1784/files#r645353416) (though it was on the wrong place)
> I could add an include call here in the test cases but I don't know if that is needed

I think that would be a bit over the top because the SEM Scan Interface already tests multiple nested calls. Or do you think that will add a lot of value?

Reply:
Sorry, I hadn't noticed there is one `!include` already. I think this "realistic" file should show more of the usage of `!include` and `!extend`. Anyway, there shouldn't be any comment like "Add via !include" or "add via !extend", and actually it's just normal text. => really do it.